lib/ast.js

```diff
@@ -345,7 +345,7 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
 var AST_Expansion = DEFNODE("Expansion", "expression", {
     $documentation: "An expandible argument, such as ...rest, a splat, such as [1,2,...all], or an expansion in a variable declaration, such as var [first, ...rest] = list",
     $propdoc: {
-        expression: "AST_Symbol the thing to be expanded"
+        expression: "[AST_Symbol] the thing to be expanded"
```

Comment:
It's not necessarily an array for that matter - it could also be an iterable.
```
$ echo "var m = new Set(); m.add(3); m.add(5); console.log(...m);" | node
3 5
```